src/routes.js

```diff
@@ -127,6 +127,57 @@ export default [
       feature: "Session"
     }
   },
+  {
```

Comment:
The whole routing schema looks excessive. :/ But let's leave it for now.

Reply:
Agree, needed by current approach, but we can get rid off at refactor.